src/browser.coffee

```diff
@@ -26,10 +26,8 @@ return unless window?
 # Ported from https://developer.mozilla.org/en-US/docs/DOM/window.btoa
 if btoa? and JSON? and unescape? and encodeURIComponent?
```

Comment:
I removed the `unescape` and `encodeURIComponent` checks as they are no longer used for the inline sourcemaps. As far as I know these functions simply undo each other so I am not sure why they were there in the first place.

I looked through other libraries that provide inline sourcemaps and none of them do this.